app/seed-data/formula_values/scripts/utils.py

```diff
@@ -0,0 +1,107 @@
+import csv
+import hashlib
+from pathlib import Path
+import random
+import sys
+import uuid
+
+
+def get_filename():
+    """returns the file name"""
+    return Path(sys.argv[0]).name
+
+
+def string_to_hash(string):
+    """generate reproducible hash from a string
+
+    Parameters
+    -----------
+    string: str
+        string you want to hash
+
+    Returns
+    --------
+    hash: int
+        a sha256 hash of the string
+
+    Example
+    ---------
+    hash_value = string_to_hash('oef')
+    # ans: 41573739547895909249334528024575358810890736899293161579861901619623096988500
+    """
+    hasher = hashlib.sha256()
+    hasher.update(string.encode("utf-8"))
+    return int(hasher.hexdigest(), 16)
+
+
+def set_seed(seed_value):
+    """sets seed of random number generator"""
+    random.seed(seed_value)
+
+
+def uuid_generate_v4():
+    """generate a version 4 UUID"""
+    return uuid.UUID(int=random.getrandbits(128), version=4)
+
+
+def uuid_generate_v3(name, namespace=uuid.NAMESPACE_OID):
+    """generate a version 3 UUID from namespace and name"""
+    assert isinstance(name, str), "name needs to be a string"
+    assert isinstance(namespace, uuid.UUID), "namespace needs to be a uuid.UUID"
+    return str(uuid.uuid3(namespace, name))
```

Comment:
I think we have these functions repeated quite a bit in different data import scripts for the seed data. Maybe worth moving this file into a shared folder, e.g. `app/seed-data/shared`.
Since these are quite short it's not too bad though.

Same for the write dic to csv and make dir functions.

Reply:
This is a good point, maybe we change this over if these get larger?